R/add_stylecss.R

```diff
@@ -2,14 +2,20 @@
 #' @description Add PHS template phs_style.css file to chosen directory.
 #'
 #' @param path String: path to add phs_style.css file. If left blank, RStudio will prompt the user.
+#' @param shinycss Logical: Whether to append shiny CSS code to the file (TRUE) or not (FALSE). If left blank, RStudio will prompt the user.
 #'
 #' @return NULL - Adds phs_style.css file to the directory.
 #' @export
 #' @examples
 #' \dontrun{
 #' add_stylecss()
 #' }
-add_stylecss <- function(path = rstudioapi::selectDirectory(caption = "Select folder to add phs_style.css")) {
+add_stylecss <- function(path = rstudioapi::selectDirectory(caption = "Select folder to add phs_style.css"),
+                         shinycss = rstudioapi::showQuestion(
+                           title = "Shiny CSS",
+                           message = "Do you want to append shiny CSS code to the file?",
+                           ok = "Yes", cancel = "No"
+                         )) {
```

Comment:
Rather than bundling the `showQuestion` into the function definition, I think it would be better to have a default (not sure if TRUE or FALSE would be best), and then do the question ask in the function body. This also means it will work non-interactive (not sure why that's needed 😆).
```suggestion
add_stylecss <- function(path = rstudioapi::selectDirectory(caption = "Select folder to add phs_style.css"),
                         shinycss = FALSE) {

if (missing(shinycss) && interactive()) {
shinycss <- rstudioapi::showQuestion(
                           title = "Shiny CSS",
                           message = "Do you want to append Shiny CSS code to the file?",
                           ok = "Yes", 
                           cancel = "No"
                         )
                         }                                    
```

Reply:
Works well when I've tested - rearranged/tidied the code a bit.